README.md

```diff
@@ -1,2 +1,62 @@
-# Hello-Microverse
-A "Hello Microverse" project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Hello Microverse project is the first solo project that Micronauts have to do. It's introduced on the second day of the program. This project teaches Git, GitHub, GitHub Flow, and linters - all in order to familiarize with web development best practices such as good coding habits and repository professionalism from early on!
+
+
+## Built With
+
+- HTML, CSS
+- Linters: Lighthouse, Webhint, Stylelint
+
+
+## Live Demo
+
+N/A
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Prerequisites
+
+Code editor and terminal.
+
+### Setup
+
+Have all necessary npm installations ready.
+
+### Install
+
+Follow linters installations [Here](https://github.com/microverseinc/linters-config/tree/master/html-css)
+
+
```

Comment:
Please your ReadMe file needs to be descriptive. kindly follow Microverse [Template](https://github.com/microverseinc/readme-template), and customize it to your project.